docs/en/observability/cloud-monitoring/aws/monitor-aws-cloudwatch-firehose.asciidoc

```diff
@@ -0,0 +1,270 @@
+
+IMPORTANT: AWS PrivateLink is not supported. Make sure the deployment is on AWS, because the Amazon Data Firehose delivery stream connects specifically to an endpoint that needs to be on AWS.
+
+[discrete]
+[[firehose-cloudwatch-step-one]]
+== Step 1: Install AWS integration in {kib}
+
+. In {kib}, navigate to *Management* > *Integrations* and browse the catalog to find the AWS integration.
+
+. Navigate to the *Settings* tab and click *Install AWS assets*.
+
+[discrete]
+[[firehose-cloudwatch-step-two]]
+== Step 2: Select a CloudWatch log group to monitor 
+
+In this tutorial, we will collect application logs from an AWS Lambda-based app and forward them to Elastic.
+
+If you already have a Lambda function, or any other service or application that logs on a CloudWatch log group, you can skip this section. Take note of the log group from which you want to collect log events and move to the next section.
+
+Otherwise, let's create a lambda function.
+
+[discrete]
+[[firehose-cloudwatch-step-two-overview]]
+=== Overview
+
+In this tutorial, we will write a simple AWS Lambda-based app, collect its application logs, and forward them to Elastic. 
+
+Like many other services and platforms in AWS, Lambda functions natively log directly to CloudWatch out of the box. Lambda functions are a great tool for experimenting on AWS.
+
+[discrete]
+[[firehose-cloudwatch-step-two-create-lambda]]
+=== Create a Lambda function
+
+1. Visit the AWS web console and open the AWS Lambda page.
+2. Click on **Create function** and select the option to create a function from scratch.
+3. Select a **Function name**
+4. As a **Runtime**, select a recent version of Python (for example, Python 3.11).
+5. Select your **Architecture** of choice between `arm64` and `x86_64`.
+6. Confirm and create the Lambda function
+
+When AWS completes the creation of the function, visit the **Code source** section and paste the following Python code as function source code:
+
+[source,python]
+----
+import json
+
+
+def lambda_handler(event, context):
+    print("Received event: " + json.dumps(event))
+----
+
+[IMPORTANT]
+=====
+Click on **Deploy** to deploy the changes to the source code.
+=====
+
+[discrete]
+[[firehose-cloudwatch-step-two-genereate-sample-logs]]
+=== Generate some sample logs
+
+With the function ready to go, we can invoke it a few times to generate sample logs.
+
+On the function page,
+
+- Select **Test**
+- Select the option to create a new test event
+- Name the test event and **Save** the changes.
+- Click on the **Test** button to execute the function.
+
+Visit the function's log group. Usually, the AWS console offers a handy link to jump straight to the log group it created for this function's logs.
+
+You should see something similar:
+
+image::firehose-cloudwatch-sample-logs.png[CloudWatch log group with sample logs]
+
+Take note of the log group name for this Lambda function, and move to the next section.
```

Comment:
IMO it's rarely necessary to tell users to move to the next section unless there is some reason they might not realize they need to perform additional steps. I would remove the extra words here and in other places (unless for some reason users are likely to be confused).